fastlane/lib/fastlane/actions/changelog_from_git_commits.rb

```diff
@@ -10,7 +10,11 @@ def self.run(params)
           UI.success("Collecting the last #{params[:commits_count]} Git commits")
         else
           if params[:between]
-            from, to = params[:between]
+            if params[:between].include?(",") # running from shell
+              from, to = params[:between].split(",", 2)
+            else
```

Comment:
And depending on how the above is structured.. we may need to add a check to make sure that this is an array?